### ISSUE TITLE
Color Scheme used in readme images

### ISSUE THREAD

**Issue by cvoege** [borela/naomi#6]
Hey @borela! I like what you're doing here especially since there is no good Flowtype syntax highlighting for Sublime. One question I have is what is the color scheme you're using in the readme? If I go into the color schemes for Naomi there's just Candyman and it looks nothing like that on my machine.

Thank you!

**Comment by borela** [comment]
Hi @cvoege could you post a picture of what i looks like now? I did some changes to it in the last commits but it could a bug if it looks too different. I did remove the "bold" style from most words as it was getting pretty confusing on things like `var.subvar.subsubvar`.

Most of the scopes right now are stable and I am thinking of converting other color schemes to scss, If you would like I'be happy to accept a pull request of your favorite color scheme, you can see the scopes in [here](https://github.com/borela/naomi/tree/master/scheme/candyman) and the [root file](https://github.com/borela/naomi/blob/master/Candyman.scsscheme), it's normal css and I use [CSScheme](https://github.com/FichteFoll/CSScheme) to compile it to tmTheme.

**Comment by cvoege** [comment]
Here you go @borela !

![screenshot - 11012016 - 09 31 28 am](https://cloud.githubusercontent.com/assets/6777709/19891354/136cb39c-a016-11e6-92a5-7ee3d0d197c9.png)

Basically the background is extremely blue so everything else becomes unreadable. I have the chance to provide some more info now so I'll do that.

I'm using Boxy Theme but the issue remains even if I disable that. 

Here's the full list of packages I have installed:

		"All Autocomplete",
		"Babel",
		"Boxy Theme",
		"Color Highlighter",
		"Duplicate Lines",
		"Naomi",
		"Package Control",
		"PlainTasks",
		"Sass",
		"SideBarEnhancements",
		"SublimeLinter",
		"SublimeLinter-contrib-eslint",
		"SublimeLinter-flow",
		"SublimeLinter-rubocop",
		"TypeScript"

Sublime Text, Unregistered, Stable Channel, Built 3126.

Linux cvoege-G55VW 4.4.0-45-generic #66~14.04.1-Ubuntu SMP Wed Oct 19 15:05:38 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

I may try next to completely uninstall sublime and reinstall.

**Comment by cvoege** [comment]
Woah that's really weird. When I take a screenshot of it and post it it looks really different. On my screen before take a screenshot it's a deep blue that clashes with the background

**Comment by borela** [comment]
I'll try to install the other packages you have to see if I can reproduce the problem.

**Comment by cvoege** [comment]
It seems to just be my monitor, which is weird. I can't really change the center monitor (it's a laptop) so there's not much I can do. How can I configure the background color?

**Comment by borela** [comment]
Try to change the background color of the color scheme, change the value from line 15 from `~/.config/sublime-text-3/Packages/Naomi/Candyman.tmTheme` (it should apply instantly after you save it), try something bizarre like red or pink first to confirm that the color scheme is working and is not another package giving trouble.

**Comment by cvoege** [comment]
Tried changing it to black and nothing happened

**Comment by cvoege** [comment]
So if I open up my sublime settings it's using "Packages/User/SublimeLinter/Candyman (SL).tmTheme", and every time i change it to the one in Packages/Naomi, it changes itself back within a few seconds. Though when I changed it this time it seems to have registered the change in background color.

**Comment by cvoege** [comment]
Even if I disable SublimeLinter it keeps doing that -.-

**Comment by borela** [comment]
Oooooooo I remember something similar in the past, edit the (SL) file, the linter creates a new file for each color scheme you use and it should update when you change the original one but I did have problems with it not updating.

**Comment by cvoege** [comment]
Yeah seems like SublimeLinter is just inconsistent about doing it. Makes sense.

Are there any plans to incorporate some other color schemes? How difficult would it be to bring an existing color scheme into Naomi? If it's not too bad I may take a crack at it in my spare time

**Comment by borela** [comment]
Yeah, I miss some color schemes like Dracula, Bittersweet... and I like to change them from time to time. 

It's easy to create them if you use SCSS or SASS, but you'll need CSScheme to compile the files to tmTheme. I just added a file to the root of the package Candyman.scsscheme and partials inside `scheme/candyman` (the files are very small).

So for example, to convert the dracula color scheme, I would extract the scopes(scopes are like classes in css) from the tmTheme to a scss file and start hacking it.

**Comment by cvoege** [comment]
So with most syntaxes, like babel-sublime for example, you can just use whatever color scheme you want (like the default monokai) and it will work out of the box. Is there a reason why this doesn't work for Naomi?

**Comment by borela** [comment]
Naomi generates very specific scopes to allow tweaking so you have things like:

* entity.name.function
* entity.name.class
* entity.name.interface
* entity.name.attribute
* entity.name.tag
* ...
Check the [flow delimiters](https://github.com/borela/naomi/blob/master/scheme/candyman/flowtype/_delimiter.scss). Most color schemes just use very vague scopes like `storage`, [monokai is extremely simple](https://github.com/cj/sublime/blob/master/Color%20Scheme%20-%20Default/Monokai.tmTheme) but on the other hand, Boxy Monokai (comes with the boxy theme) works a lot better.

**Comment by cvoege** [comment]
![screenshot - 11012016 - 11 24 58 am](https://cloud.githubusercontent.com/assets/6777709/19895104/d90e923c-a025-11e6-9b2a-155445f4aa7f.png)

Boxy Monokai with Naomi looks like this for me. Any Ideas?

**Comment by borela** [comment]
It doesn't apply colors to `entity.name.attribute`, so things like `className` are using the foreground.

**Comment by borela** [comment]
Looking at the [source](https://github.com/oivva/st-boxy/blob/dev/schemes/Boxy%20Monokai.tmTheme), line 272 `entity.other.attribute-name` just change that to `entity.name.attribute` and it should show the color expected.

That's the other issue I encountered in the past, the scopes were hard to predict, very inconsistent.

**Comment by cvoege** [comment]
Relevant: https://xkcd.com/927/

**Comment by borela** [comment]
LOL I do agree and I did use most the [scopes](https://www.sublimetext.com/docs/3/scope_naming.html) in the manual, the minor deviations happened during the writing of the syntaxes, but don't worry I am only writing syntaxes that don't exist or the existing ones are broken.

I could try to put the missing scopes back, I would need someone to list them for me(the syntaxes are really big).

But these problem with inconsistencies comes way back when I tried to use a color scheme and while it worked great on a language, it looked horrible on another so, the easiest route is to actually convert them to SCSS so that the bugs(from other syntaxes too) could fixed.

Boxy ocean with minor tweaks.
![captura de tela 2016-11-01 13 41 20](https://cloud.githubusercontent.com/assets/11317458/19895761/1f92b306-a039-11e6-865d-c1ed014d2fcc.png)
